### PR TITLE
Bug/form

### PR DESCRIPTION
**Pull request by Billwilke42**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

-  Form bug, react icon, no spaces after ifs, not deployed on heroku

## What is the new behavior?

- Form bug fixed, adds book icon, adds spaces after ifs, deployed on heroku

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Form/Form.js [hunk 1]
```diff
@@ -27,9 +27,14 @@ class Form extends React.Component {
...
-      if(prevState.isDisabled === this.state.isDisabled) {
-        this.setState({ isDisabled: false })
+    if (this.state.name && this.state.round1 && this.state.round2 && this.state.round3 && this.state.round4 && this.state.round5 && this.state.round6) {
+      if (prevState.isDisabled) {
+        this.setState({ isDisabled: false }) 
+      } 
+    } 
+    if (!this.state.name || !this.state.round1 || !this.state.round2 || !this.state.round3 || !this.state.round4 || !this.state.round5 || !this.state.round6) {
+      if(!prevState.isDisabled) {
+        this.setState({ isDisabled: true })
```
Comment (KyleWong2510) [inline src/Form/Form.js:37]: tricky tricky

File: src/thunks/getTeachersPets.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+  const url = 'https://chalkboard-trivia.herokuapp.com/api/v1/scores'
 
   return async (dispatch) => {
     try {
       const response = await fetch(url)
-      if(!response.ok) {
+      if (!response.ok) {
         throw Error(response.statusText)
       }
       const data = await response.json()
```
Comment (KyleWong2510) [inline src/thunks/getTeachersPets.js:12]: glad we got it to deploy!

File: public/index.html [hunk 3]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Chalkboard Trivia</title>
```
Comment (lilianaweimer) [inline public/index.html:27]: Nice fix!

File: src/Form/Form.js (hunk 1 above)
Comment (lilianaweimer) [inline src/Form/Form.js:37]: Glad we got this bug squashed!

**Pull request merged**